go/vt/proto/tabletmanagerdata/tabletmanagerdata.pb.go

```diff
@@ -5,8 +5,9 @@ package tabletmanagerdata
 
 import (
 	fmt "fmt"
-	proto "github.com/golang/protobuf/proto"
 	math "math"
+
+	proto "github.com/golang/protobuf/proto"
```

Comment:
this generated file should not have changed since you didn't change the .proto file. Can you run `goimports` on it and see if the diff goes away?

Reply:
git complains about it when I was trying to commit after I have rebased my branch. So I ran the goimports on it. I can revert this file and push it again.

Reply:
Not able to revert this change, its not letting me push unless I run gomport on this file.

Reply:
Then go ahead and commit it. It must have been changed by someone else.